jumanji/training/networks/connector/actor_critic.py

```diff
@@ -50,8 +50,11 @@ def make_actor_critic_networks_connector(
     parametric_action_distribution = MultiCategoricalParametricDistribution(
         num_values=num_values
     )
+    num_agents = num_values.shape[0]
+    num_actions = num_values[0]
```

Comment:
NIT: maybe add an inline comment on what the shape, contents of num_values is so it is more intuitive

Reply:
Done :heavy_check_mark: